### PR TITLE
Feat: dawnport revision and fixes

### LINKED ISSUE #1074: Dawnport com bug nas portas e impossibilidade de ir pra main
Como as portas nao funcionam e so tem uma porta, ao ir pro barco o NPC diz que ja tem uma vocacao e impossibilita o embarque a main

### PR DESCRIPTION
**Pull request by dudantas**
With this pull request, dawnport is running 100%, being set as town 1
Rookgaard is secondary, as it is in the global, access will be through dawnport, in the NPC inigo.
All npcs have been revised, everything is working as globally (as far as possible)

Added: 
Added Dawnport quest
Co-Authored-By: Billfabio <billfabio@users.noreply.github.com>
Added Legion helmet quest
Co-Authored-By: Billfabio <billfabio@users.noreply.github.com>
Added Vocation trial script (only the conversion of skills is missing)
Added Dawnport npcs spells sellers
Co-Authored-By: Billfabio <billfabio@users.noreply.github.com>
Added function to load the Book and Sign table
Added dawnport books and signs
Added tutorial steps (1 and 2)
Added script to send the player who gets level 20 to the temple after 5 minutes
Added first dawnport items on login.lua
Added update samples in migration and schema.sql, it will be corrected to work with dawnport
Added the script for the dawnport lever
Added Sacred Snake loot
Added sale of items at NPC Richard
Added message of when level 8 is reached
Added distance effect for mage weapons

Created four new vocations for dawnport.
The dawnport vocations have rooker status, that is, they have a fixed gain of health/mana/cap and other attributes up to level 8. Therefore, proper vocations for dawnport are necessary.
As a result of this change, an enum was created with vocation and clientids ids in the tables folder in libs.
Several calls to vocations were changed from "ids" to "clientid", since before it was bugged and only allowed use with the vocation where the id was set. That way, it will also be possible to use the calls like "elite knight", "royal paladin" and etc ...

Modified:
Defined action 40001/2/3/4 for the vocation tiles.

Refactored the function of loading the action table, it is now possible to add an action without having to check the itemId, setting the variable "itemId = false", thus allowing to add the same action in several different ids and in different positions. Notice that the action is added to the top item.
Now the Lua action/unique script will identify more than one position in the same action.
Configured to use the {x =, y =, z =}

Table of additional maps moved to the folder "tables" and the script attached to startup.lua.
Towns table moved to the "tables" folder.
Change in towns IDs
DIsabled loot loss until lvl 8 for dawnport
DIsabled adventurer blessing
Disabled standard messages on NPCS WalkAway
Inigo send correct to rookgaard

Revised:
Dawnport npcs (as well as all revised lines)
NPC Plunderpurse now limits the deposit to 1000 gps
NPC Wentworth limits the deposit to 1000 gps for lvl 8 and 2000 gps for lvl 9 to 20
NPC Black Bert
Blocked transfers from mainland bank to players in dawnport (thanks @andersonfaaria )
Changed town from rookgaard tutorial to dawnport tutorial
Fixed bugs on the map
Revised dawnport npcs (as well as all revised talks)
Co-Authored-By: Billfabio billfabio@users.noreply.github.com

Fixed some bugs in Minotaur Poacher
It will check if the player has a mainland char (via the storage it earns when leaving dawnport), and will only take it to rookgaard if it already has a mainland char via the getAccountStorage function (thanks @andersonfaaria for develop the function)

Special thanks to @Billfabio and @andersonfaaria

### CODE REVIEW

File: data/movements/scripts/dawnport/vocation_trial.lua [hunk 1]
```diff
@@ -120,12 +114,14 @@ local function changeVocation(player, fromVocation, toVocation)
...
 		}
 	}
-
+	player:setVocation(toVocation) -- first set vocation, to add items
 	for toVocation = 1, 4 do
 		for slot, info in pairs(vocationsItems[toVocation]) do
 			local itemCount = player:getItemCount(info[1])
 			if itemCount > 0 and info[3] then
-				player:removeItem(info[1], itemCount)
+				if info[1] ~= 8704 and info[1] ~= 7620 and info[1] ~= 23723 and info[1] ~= 23722 then
```
Comment (gpedro) [inline data/movements/scripts/dawnport/vocation_trial.lua]: that's for what?

Comment (dudantas) [reply data/movements/scripts/dawnport/vocation_trial.lua]: > that's for what?

the script is from @billfabio

Comment (Billfabio) [reply data/movements/scripts/dawnport/vocation_trial.lua]: Reorganization between bp items and those added when acquiring a new vocation

**Comment by dudantas** [comment]
> text error
> [[
> Personal log book of Mr Morris. (The following lines are written in a code that you cannot make out. It seems Mr Morris is a very distrustful person.)
> ]]

Can use ""

**Pull request merged**